applications/matter_bridge/Kconfig

```diff
@@ -156,8 +156,12 @@ endif
 config CHIP_ENABLE_READ_CLIENT
 	default y
 
+config NCS_SAMPLE_MATTER_PERSISTENT_STORAGE
+	default y
```

Comment:
We can, but we have more prj.conf than one. I thought it was better to keep it as one for all. Any obvious cons apart from not being transparent?

Reply:
TBH I also started to think that we a little bit overuse defaults instead of regular prj.confs, but it needs wider discussion to make some common rules. For now I think it's ok to have it as default, because as Marcin mentioned we already have got bunch of the similar Kconfigs.